spec/lita/services/market_manager_spec.rb

```diff
@@ -283,6 +292,18 @@ def setup_lunchers
         expect(lunch_assigner.winning_lunchers_list).to include(fdom.mention_name)
         expect(lunch_assigner.winning_lunchers_list).not_to include(andres.mention_name)
       end
+
+      it 'uploads transaction' do
+        subject.execute_transaction
+        expect(sh_manager).to have_received(:insert_new_row).with([
+          Time.now.strftime('%Y-%m-%d'),
+          fdom.name,
+          fdom.id,
+          andres.name,
+          andres.id,
+          1
+        ])
```

Comment:
Indent the right bracket the same as the first position after the preceding left parenthesis.